client/components/chart/style.scss

```diff
@@ -91,6 +94,31 @@
 				flex-wrap: nowrap;
 				justify-content: flex-start;
 				align-items: center;
+
+				&.key-row {
+					display: flex;
+					flex-direction: row;
+					justify-content: space-between;
+					width: 100%;
+
+					.key-container {
+						width: 100%;
+						min-width: 100px;
+
+						.key-colour {
```

Comment:
Should we use `color` instead of `colour` to be consistent with CSS properties?

Reply:
Fixed in https://github.com/woocommerce/wc-admin/pull/350/commits/70ab5a5bf0da1ec5d3a9d185a81cf11f6a68939e